.github/workflows/cache_data.yaml

```diff
@@ -0,0 +1,43 @@
+# This workflow gets and uploads the GMT data artifacts used in the PyGMT CI tests
+name: Cache data
+
+on:
+  # Uncomment the 'pull_request' line below to manually re-cache data artifacts
+  # pull_request:
+  # Schedule runs on 12 noon every Sunday
+  schedule:
+    - cron: '0 12 * * 0'
+
+jobs:
+  gmt_cache:
+    name: Cache GMT artifacts
+    runs-on: macOS-latest
+
+    steps:
+      # Setup Miniconda
+      - name: Setup Miniconda
+        uses: goanpeca/setup-miniconda@v1.6.0
+        with:
+          python-version: ${{ matrix.python-version }}
```

Comment:
```suggestion
          python-version: 3.7
```

Reply:
Actually don't need Python at all, so will remove the line.